api_server/charity/routers.py

```diff
@@ -4,36 +4,85 @@
 
 from starlette.status import HTTP_200_OK, HTTP_201_CREATED
 
-from charity.schemas import CharityInputSchema, CharityOutputSchema, CharityUpdateSchema
+from charity.schemas import CharityInputSchema, CharityOutputSchema, CharityShortDescriptionSchema, CharityUpdateSchema
 from charity.services import CharityService
 from common.schemas.responses import ResponseBaseSchema
 
 charities_router = APIRouter(prefix='/charities', tags=['Charities'])
 
 
+@charities_router.get("/{org_id}", response_model=ResponseBaseSchema, status_code=HTTP_200_OK)
+async def show_charity_organisation(org_id: UUID, charity_service: CharityService = Depends()):
+    """
+    Retrieves full info about definite organisation.
+
+    Args:
+        org_id: id of organisation.
+        charity_service: dependency as business logic instance
+
+    Returns:
+        ResponseBaseSchema object with CharityOutputSchema object as response data.
+    """
+    return ResponseBaseSchema(status_code=HTTP_200_OK,
+                              data=[
+                                  CharityOutputSchema.from_orm(await charity_service.get_exact_organisation(org_id))
+                                  ],
+                              errors=[])
+
+
+@charities_router.get("/", response_model=ResponseBaseSchema, status_code=HTTP_200_OK)
+async def show_charities_list(charity_service: CharityService = Depends()):
+    """
+    Retrieves info about active organisations.
+
+    Args:
+        charity_service: dependency as business logic instance
+
+    Returns:
+        ResponseBaseSchema object with list of CharityOutputSchema object as response data.
+    """
+    return ResponseBaseSchema(status_code=HTTP_200_OK,
+                              data=[
+                                  CharityShortDescriptionSchema.from_orm(charity)
```

Comment:
Why for /charities we using this "short" schema?

Reply:
According to my logic, we retrieve short description of each organisation like `title` and `description`. 
And after we want to know a little bit more of definite organisation, we pass to `/charities/{id}` endpoint and find out info e.g. phone number, email etc.

Reply:
Maybe it is a better idea to include in ```/charities``` all public fields? it is quite unusual to "hide" public fields, and for what reason?

Reply:
If They're not being used, why do we need to add them?

Reply:
But ok, i will add this fields, if you think it's necessary

Reply:
Usually there private and public fields, and it is hard to tell how and where they are gonna be used, therefore it is a good idea to put all public fields out there. This what they for, to be public, is it not?

Reply:
I think, it's better to ask somebody like Gosha to solve our discussion. And for now, I've added all public fields to response